go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -188,10 +188,24 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
+	aclName = strings.ReplaceAll(aclName, ":", "\\:")
+	// ACL names are limited to 63 characters
+	if len(aclName) > 63 {
+		// Add the length of the IP (max 39 with colons), plus 14 for '\\' for each ':' in IP,
+		// plus length of sourcePort (max 5 char), plus 3 for additional '\\:' to separate,
+		// plus '-' between lb and IP. With full IPv6 address and 5 char port, max ipPortLen is 62.
```

Comment:
so if the maximum length for v6 is 62 chars, and our maximum acl name is 63. Then with a full v6 address we would have something like:

`a-<ipv6>-<ip port>`

Meaning that we would have one character for a our lb name?

Reply:
For full IPv6, yes. Lots of colons which have '\\'. Are those required?

Reply:
Can we replace ":" in ipv6 by "." or "-". This will still keep the original value intact but will save 14 characters for "\\" and thus offering more bytes for lb part in the ACL name?